jenkins-integration.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+DEFAULT_PACKAGE_TIMEOUT="45m"
+
+if [ $1 == "-m" ]; then
+    echo "Running in automated master integration mode"
+    # Set automated setting parameters
+    SG_COMMIT="master"
+    TARGET_PACKAGE="..."
+    TARGET_TEST="ALL"
+    RUN_WARLUS="true"
```

Comment:
typo causing the jobs not to run the walrus tests

```suggestion
    RUN_WALRUS="true"
```